docs/getting-started-objc-macos.md

```diff
@@ -0,0 +1,151 @@
+# Getting started with macOS and Embeddinator-4000
+
+## What you will need
+
+* Follow instructions in our [Getting started with Objective-C](getting-started-objective-c.md) guide.
+
+* A .NET Assembly to use with **Embeddinator-4000**.
+
+* A macOS Cocoa Application
+
+Please continue after you have followed the instructions in our [Getting started with Objective-C](getting-started-objective-c.md) guide. If you already have a .NET Assembly you can skip directly into **Using Embeddinator-4000** section.
+
+## Creating a .NET Assembly
+
+In order to build a .NET assembly you will need to open [Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/) and create a new **.NET Library Project** by doing *File > New Solution > Other > .NET > Library* . Click next and give *Weather* as the *Project Name* and click on *Create*.
```

Comment:
by doing *File > New Solution > Other > .NET > Library*.

Whitespace! :D

...give *Weather* as the *Project Name*, then click on *Create*.